spec/import_spec.rb

```diff
@@ -44,6 +44,40 @@ def upload_file!(name, ext='csv')
     find_button('Import').click
   end
 
+  context "posts index" do
+    before do
+        Author.create!([{name: 'John', last_name: 'Doe'}, {name: 'Jane', last_name: 'Roe'}])
+        add_post_resource({
+          validate: true,
+          headers_rewrites: {
+              :'Author Name' => :author_id
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.